conftest.py

```diff
@@ -1,10 +1,15 @@
+import os
 import pytest
+import shutil
 
 from openpilot.common.prefix import OpenpilotPrefix
+from openpilot.system.hardware.hw import Paths
 
 
 @pytest.fixture(scope="function", autouse=True)
 def global_setup_and_teardown():
   # setup a clean environment for each test
   with OpenpilotPrefix():
+    os.makedirs(Paths.log_root())
```

Comment:
this should be in the prefix

Reply:
it is in the prefix (since comma_home has the prefix), but the folder isn't created automatically

Reply:
so its like /home/batman/.comma12345678/log now

Reply:
it should be handled in the prefix...

Reply:
done